model_hub/examples/huggingface/token-classification/ner_utils.py

```diff
@@ -0,0 +1,128 @@
+"""
+These functions extract parts of the code from the named-entity recognition example
+in huggingface transformers.  The source example code changes rapidly due to frequent refactorings
+but the main structure should be similar.
+See the original example at:
+https://github.com/huggingface/transformers/blob/master/examples/token-classification/run_ner.py
```

Comment:
we probably need to come up with a slightly more formal citation for code that we are mostly copying.  But if it's ok with you I'd rather deal with this comment as the absolute last detail (I'm posting it now so I don't forget)

Reply:
i added a todo to track this and will merge a separate PR into this branch to add citations.

Reply:
actually, i'm doing it by PR instead